packages/logs/src/domain/logsSessionManager.ts

```diff
@@ -19,8 +19,15 @@ export const enum LoggerTrackingType {
 }
 
 export function startLogsSessionManager(configuration: LogsConfiguration): LogsSessionManager {
-  const sessionManager = startSessionManager(configuration.cookieOptions, LOGS_SESSION_KEY, (rawTrackingType) =>
-    computeSessionState(configuration, rawTrackingType)
+  if (!configuration.sessionStoreStrategyType) {
+    throw new Error('Cannot initialize Logs Session Manager without a storage.')
+  }
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏this check is done from the caller though 🤔
what is your intent here?

Reply:
Non-null assertion for now.